File: Src/LibNoise/Generators/Constant.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LibNoise
{
    public class Constant
        : IModule
    {
        public double Value { get; set; }

        public Constant(double value)
        {
            Value = value;
        }

        public double GetValue(double x, double y, double z)
        {
            return Value;
        }
    }
}

File: js/NoiseLib/Generators/Constant.js
LibNoise.Constant = function(value)
{
  this.Value = value;
  this.Name = "LibNoise.Constant";
}

LibNoise.Constant.prototype.getInput = getNone;
LibNoise.Constant.prototype.setInput = setNone;

LibNoise.Constant.prototype.GetValue = function (x, y, z)
{
  return this.Value;
}
